Formula/gwyddion.rb

```diff
@@ -3,6 +3,7 @@ class Gwyddion < Formula
   homepage "http://gwyddion.net/"
   url "http://gwyddion.net/download/2.50/gwyddion-2.50.tar.gz"
   sha256 "f3834dae31d9bf696e8d59e2aa79a373a30d5f6caa6033601d2f9d57afa154f3"
+  revision 1
```

Comment:
This formula seems to require further changes. Probably just shunting to `python@2` here.

```
checking for a Python interpreter with version >= 2.4... python2
checking for python2... /usr/local/opt/python/bin/python2
checking for python2 version... 2.7
checking for python2 platform... darwin
checking for python2 script directory... ${prefix}/lib/python2.7/site-packages
checking for python2 extension module directory... ${exec_prefix}/lib/python2.7/site-packages
checking size of pid_t... 4
checking for python2.7-config... /usr/local/opt/python/bin/python2.7-config
checking if we can link a test Python program... yes
```